SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -224,19 +225,15 @@
     <!-- **************************************************************************** -->
     <!-- Create the project-specific directories for conf and out -->
 
-    <!-- Calculate a project-specific directory names for conf and out -->
+    <!-- Create folder with unique name in the .sonarqube/out/ folder -->
+    <MakeUniqueDir Path="$(SonarQubeOutputPath)" Prefix="$(MSBuildProjectName)_">
+      <Output TaskParameter="UniquePath" PropertyName="ProjectSpecificOutDir" />
+      <Output TaskParameter="UniqueName" PropertyName="UniqueName" />
+    </MakeUniqueDir>
+    <!-- Create .sonarqube/conf/ sub-folder with the same unique name -->
     <PropertyGroup>
-      <FolderDisambiguator Condition=" $(FolderDisambiguator)=='' ">_$([System.DateTime]::Now.ToString("ffff"))</FolderDisambiguator>
```

Comment:
@duncanp-sonar do you think we should keep the FolderDisambiguator setting for backwards compatibility here?

Reply:
No, we shouldn't need it.